dashboard/api.py

```diff
@@ -331,6 +331,38 @@ def format_courserun_for_dashboard(course_run, status_for_user, certificate=None
     return formatted_run
 
 
+def update_cached_enrollment(user, enrollment, course_id, now):
```

Comment:
do you think it is worth decorate with a `transaction.atomic`? I mean, if it is used by itself I still would like to change the DB atomically